flang/test/Examples/hello.f90

```diff
@@ -1,13 +1,12 @@
-! Note: The Fortran runtime libraries have dependences on the C++ runtime and
-! LLVM libraries.  To work around the former, this test explicitly links in
-! libstdc++.a.  To work around the latter, the source of Common/enum-set.h was
+! Note: The Fortran runtime libraries have dependences on LLVM libraries.
+! To work around this issue, the source of Common/enum-set.h was
```

Comment:
We should double check that this note about enum-set.h is still true. I seem to recall Peter K. having fixed it. I don't recall now, but I guess it relates to the `-L%L` command line flag.

Reply:
Good point, the comment is outdated, `enum-set.h` was fixed in llvm in https://github.com/llvm/llvm-project/commit/7cccd49a553b6381f57f0ad8545fd5ea9e329afd. I removed it. I verified that fir-dev and llvm enum-set.h are byte to byte the same now.

The  `-L%L` is not related to this, it is providing the library path for `-lFortranRuntime -lFortranDecimal` and is required.